pkg/syncer/syncer.go

```diff
@@ -251,30 +223,13 @@ func (s *syncer) Stop() {
 }
 
 // UpdateTarget updates target and returns wether did update or not.
-func (s *syncer) UpdateTarget(projectID, instanceID string, credentials *Credentials) bool {
-	updated := false
-
-	if s.projectID != projectID {
-		updated = true
-		s.projectID = projectID
-	}
-
-	if s.instanceID != instanceID {
-		updated = true
-		s.instanceID = instanceID
-	}
```

Comment:
These checks are not needed because `Spec.TargetInstance` is immutable - https://github.com/mercari/spanner-autoscaler/blob/master/api/v1beta1/spannerautoscaler_webhook.go#L116